packages/example-app-base/src/utilities/data.ts

```diff
@@ -30,11 +30,12 @@ export function createListItems(count: number, startIndex = 0): any {
 
 export function createGroups(
   groupCount: number, groupDepth: number, startIndex: number,
-  itemsPerGroup: number, level?: number, key?: string) {
-  key = key ? key + '-' : '';
-  level = level ? level : 0;
+  itemsPerGroup: number, level: number = 0, key: string = '') {
+  if (key !== '') {
```

Comment:
This change here wasn't mechanical, would appreciate another set of eyes looking at this closely to make sure I didn't mess it up.